File: apiForMagazyn/src/main/java/org/apiForMagazyn/api/prod/Prod.java
package org.apiForMagazyn.api.prod;

import lombok.Getter;
import lombok.Setter;
import org.apiForMagazyn.api.adres.Adres;

import javax.persistence.*;
import javax.validation.constraints.Size;

@Entity
@Table
@Setter
@Getter
public class Prod {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idProd;

    @Size(max = 50)
    private String name;

    @Size(max = 50)
    private String person;

    @Size(max = 20)
    private String tel;
    @OneToOne
    @JoinColumn(name="idAd")
    private Adres adres;
}

File: Magazyn/Magazyn/Tables/Prod.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Magazyn.Tables
{
    class Prod
    {
        public int idProd { set; get; }
        public string name { set; get; }
        public string person { set; get; }
        public string tel { set; get; }
        public Adres adres { set; get; }
    }
}
